homeassistant/components/vorwerk/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "domain": "vorwerk",
+  "name": "Vorwerk Kobold",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/vorwerk",
+  "requirements": [
+    "pybotvac==0.0.20"
+  ],
+  "codeowners": [
```

Comment:
Are @dshokouhi and @Santobert planning to also be codeowners of this integration? If not we should remove them.

Reply:
I copied the neato component for switch.py, sensor.py and vacuum.py.
But I will remove them. :)